news/2022-08-25-changes-to-osu-taiko-star-rating-part-1.md

```diff
@@ -0,0 +1,68 @@
+
+In Part 1 of this osu!taiko star rating rework, we have re-evaluated the Stamina and Colour components of difficulty, with the capacity for changes following community feedback. In addition, the performance points system has been adjusted to bring it closer in line with community expectations. During this time, we have been and will continue to work on other difficulty components (small and large scale) in preparation for future releases.
+
+Moving forward, osu!taiko star rating will be in a constant state of flux, particularly over the next year. This is generally desirable as we would want difficulty to accurately reflect the latest trends, but we understand and apologise if this may bring some unintended complications or frustrations, especially in the short term.  We are fully aware that this rework is not a complete solution in its current state.
+  
+## Release 
+- ☑ a bunch of checkboxes.
+  
+## Summary of changes
+
+### Stamina
+- Old stamina system assumes a two finger KDDK full-alt playstyle, with each hand being calculated separately.
+- New stamina system calculates the minimum stamina requirement theoretically needed for any pattern with 4 input keys (it assumes all 4 keys are independent with no handedness attached).
+- This does not accurately represent any single playstyle - this is desirable because we cannot feasibly detect playstyles. 
+- This "minimum" of sorts is therefore the fairest judgement of difficulty - any additional difficulty arising from specific patterning is attached to the playstyle itself.
+
+### Colour
+- Old colour system detected the frequency of colour changes and had penalties for specific repeated patterns.
+- New colour system evaluates all patterns using a compression algorithm, and adds those patterns to a dictionary.
+- The more information that is needed to describe a pattern, the harder it is.
+- Maps that have a larger dictionary (ie. use a large variety of patterns) will be evaluated as being harder.
+- The dictionary also has a "memory" of the patterns used. If a known pattern reappears later in the map, its difficulty will scale with a function of how long it last appeared in the map (ie. close repeated patterns are nerfed).
+
+### PP rework
+- osu!taiko PP has historically been calculated by summing the "Accuracy PP" and "Difficulty PP" components of a score. 
+
+#### Global Changes
+The global multiplier affects the final pp value of the map (ie. it is applied at the end of pp calculation).
+- Reduce the HD global multiplier (to increase the gap between EZHD and HD only scores)
+- Reduce the EZ global multiplier (to increase the gap between EZ and NM scores)
```

Comment:
The lack of any quantifier of the reduction is bothering me here. There's no indication of how severe the changes are here (5%? 50%?)

Reply:
Applied in b824956